.changeset/early-dancers-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+"@mysten/sui.js": patch
+---
+
+Update ts-sdk e2e test to reflect new rpc error language
```

Comment:
Probably don't even need a patch for this

Reply:
GitHub complained :0